### PR TITLE
Implement player attack animation

### PR DESCRIPTION
**Pull request by NicolasLaure**

### CODE REVIEW

**Comment by NicolasLaure** [comment]
[[BUG] [PLAYER] attack feeling](https://trello.com/c/OL37aFMM/225-bug-player-attack-feeling)

**Comment by NicolasLaure** [comment]
[[Player] Implement player attack animation](https://trello.com/c/isJDpqKO/241-player-implement-player-attack-animation)

File: Assets/Scripts/LevelManagement/Sequences/StartLevelSequence.cs [hunk 1]
```diff
@@ -68,9 +70,11 @@ private IEnumerator MovePlayerToMiddle()
                 player.transform.position += new Vector3(0, 0, playersVelocity) * Time.deltaTime;
                 yield return null;
             }
-            
+
             PlayerAgent agent = player.GetComponent<PlayerAgent>();
             agent.EnableFSM();
+            PlayerAttack playerAttack = player.GetComponent<PlayerAttack>();
+            playerAttack.enabled = true;
```
Comment (Maxi-F) [inline Assets/Scripts/LevelManagement/Sequences/StartLevelSequence.cs:77]: Maybe we should think about an event to handle this stuff

File: Assets/Scripts/Player/Animation/PlayerAnimationHandler.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    public void StartAttackAnimation()
+    {
+        _animator.SetTrigger("Attack");
+    }
+
+    public void SetAttackProgress(float progress)
+    {
+        _animator.SetFloat("attackProgress", progress);
```
Comment (Maxi-F) [inline Assets/Scripts/Player/Animation/PlayerAnimationHandler.cs:31]: Strings should be properties

**Review by Maxi-F — APPROVED** [review]

**Pull request merged**